### PR TITLE
Fix linter

### PR DESCRIPTION
**Pull request by lihannan99**
Currently we can't do `make bins` due to there are more than 40 lint issues.
This is just address those lint issues.

### CODE REVIEW

File: internal/activity.go [hunk 1]
```diff
@@ -269,6 +270,13 @@ func WithActivityTask(
 	} else {
 		deadline = startToCloseDeadline
 	}
+
+	logger.With(
+		zapcore.Field{Key: tagWorkflowType, Type: zapcore.StringType, String: *task.WorkflowType.Name},
+		zapcore.Field{Key: tagWorkflowID, Type: zapcore.StringType, String: *task.WorkflowExecution.WorkflowId},
+		zapcore.Field{Key: tagRunID, Type: zapcore.StringType, String: *task.WorkflowExecution.RunId},
+	)
```
Comment (dmetzgar) [inline internal/activity.go]: Is this logging change intended to go with this PR?

File: internal/common/metrics/capturingStatsReporter.go [hunk 2]
```diff
@@ -57,80 +60,98 @@ type CapturingStatsReporter struct {
...
 func (c *CapturingStatsReporter) Gauges() []CapturedGauge {
 	return c.gauges
 }
 
+// Counts return Counts
 func (c *CapturingStatsReporter) Counts() []CapturedCount {
 	return c.counts
 }
 
+// CapturedCount has accociated name, tags and value
```
Comment (dmetzgar) [inline internal/common/metrics/capturingStatsReporter.go]: associated

File: internal/common/metrics/capturingStatsReporter.go [hunk 3]
```diff
@@ -57,80 +60,98 @@ type CapturingStatsReporter struct {
...
 func (c *CapturedCount) Tags() map[string]string {
 	return c.tags
 }
 
+// Name return the name of CapturedCount
 func (c *CapturedCount) Name() string {
 	return c.name
 }
 
+// CapturedGauge has capturedGauge name, tag and values
```
Comment (dmetzgar) [inline internal/common/metrics/capturingStatsReporter.go]: Does the second capturedGauge need to be capitalized?

File: internal/internal_workflow_test.go [hunk 4]
```diff
@@ -621,7 +621,7 @@ func receiveAsync_CorruptSignalOnClosedChannelWorkflowTest(ctx Context) ([]int,
 	return result, nil
 }
 
-func receiveAysnc_CorruptSignalWorkflowTest(ctx Context) ([]message, error) {
+func receiveAysncCorruptSignalWorkflowTest(ctx Context) ([]message, error) {
```
Comment (dmetzgar) [inline internal/internal_workflow_test.go]: Fix spelling of Async

File: internal/activity.go (hunk 1 above)
Comment (lihannan99) [reply internal/activity.go]: No, It was planning go with this pr: https://github.com/uber-go/cadence-client/pull/669
I was somehow checkout out along with this commit.

Comment (lihannan99) [reply internal/activity.go]: I will remove this commit

**Pull request merged**